content/docs/configuration/issuers.md

```diff
@@ -15,6 +15,7 @@ The following list contains all known cert-manager issuer integrations.
 | 🥈   | aws-privateca-issuer        | [📄][config:aws-privateca-issuer]   | [AWS Private Certificate Authority][ca:aws-privateca-issuer]           | -                                                 | [✔️][release:aws-privateca-issuer]   | ✔️              |
 | 🥈   | ca-issuer (in-tree)         | [📄][config:ca-issuer]              | CA issuer                                                              | -                                                 | [✔️][release:cert-manager]           | ✔️              |
 | 🥈   | command-issuer              | [📄][config:command-issuer]         | [Keyfactor Command][ca:command-issuer]                                 | -                                                 | [✔️][release:command-issuer]         | ✔️              |
+| 🥈   | cview-issuer                | [📄][config:cview-issuer]           | [cview-issuer][ca:cview-issuer]                                        | -                                                 | [✔️][release:cview-issuer]           | ❌              |
```

Comment:
Do you have a releases page? We have to make sure the issuer has had a release in the last 12 months.

Also, can you confirm the following (does not require a proof):
- These issuers are known to support and honor [approval](https://cert-manager.io/docs/concepts/certificaterequest/#approval).

Reply:
There is a release page for helm chart. But the release of issuer image is in a private repo.
Images for issuer are visible on docker hub . https://hub.docker.com/r/devsecurely/cview-issuer/tags